app/views/hyrax/my/_collection_action_menu.html.erb

```diff
@@ -14,20 +14,23 @@
         <%= t("hyrax.dashboard.my.action.view_collection") %>
       <% end %>
     </li>
-    <li role="menuitem" tabindex="-1">
-      <%= link_to hyrax.edit_dashboard_collection_path(id),
-                  class: 'itemicon itemedit',
-                  title: t("hyrax.dashboard.my.action.edit_collection") do %>
-        <%= t("hyrax.dashboard.my.action.edit_collection") %>
-      <% end %>
-    </li>
-    <li role="menuitem" tabindex="-1">
-      <%= link_to hyrax.dashboard_collection_path(id),
-            class: 'itemicon itemtrash delete-collection-button',
-            title: t("hyrax.dashboard.my.action.delete_collection") do %>
-        <%= t("hyrax.dashboard.my.action.delete_collection") %>
-      <% end %>
-    </li>
+    <% if can? :edit, collection_presenter.solr_document %>
+      <li role="menuitem" tabindex="-1">
+        <%= link_to hyrax.edit_dashboard_collection_path(id),
+                    class: 'itemicon itemedit',
+                    title: t("hyrax.dashboard.my.action.edit_collection") do %>
+          <%= t("hyrax.dashboard.my.action.edit_collection") %>
+        <% end %>
+      </li>
+      <li role="menuitem" tabindex="-1">
+        <%= link_to hyrax.dashboard_collection_path(id),
+              class: 'itemicon itemtrash delete-collection-button',
+              title: t("hyrax.dashboard.my.action.delete_collection") do %>
+          <%= t("hyrax.dashboard.my.action.delete_collection") %>
+        <% end %>
+      </li>
+    <% end %>
+    <%# TODO: Only managers and depositors can add collections to collections. %>
```

Comment:
I added this as a TODO for now. It looks like `Hyrax::Collections::PermissionService` has a method stubbed that can handle this.